docs/source/learn/examples.md

```diff
@@ -3,14 +3,11 @@
 
 TODO add "core" notebooks:
 
-- https://docs.pymc.io/en/stable/pymc-examples/examples/generalized_linear_models/GLM-linear.html
-  - https://github.com/pymc-devs/pymc-examples/blob/main/examples/generalized_linear_models/GLM-linear.ipynb
+- {ref}`nb:GLM-linear`
```

Comment:
```suggestion
- {ref}`GLM-linear`
```

there are a handful of notebooks in the versioned docs that need to be referenced without the `nb:` as they are not part of the example gallery website